packages/scrum-client/src/content/VotingCards/VotingCards.js

```diff
@@ -20,7 +20,9 @@ function getPlanningSession(setCardDeckParam, roomId, setSessionInformation) {
   try {
     PlanningSessionAPI.get(roomId).then(data => {
       setSessionInformation(data);
-      setCardDeckParam(DECKS.byLabels(data.cardDeck).values);
+      if (DECKS.byLabels(data.cardDeck) !== undefined) {
```

Comment:
Why "DECKS.byLabels(data.cardDeck) !== undefine" is a correct evaluation ? 
This component should always return a value , righ ?  if not , it is just hiding another error . I am right?

Reply:
It was related to babel you are right